CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Fixed
+- Removed extra quotation mark for TDG jobs_average_panel query (PR #221)
```

Comment:
```suggestion
- Removed extra quotation mark for TDG jobs_average_panel query (#220)
```

We link tickets if there is a ticket and PR in case there is none.